README.md

```diff
@@ -20,6 +20,9 @@
     # Checkout git repositories
     git clone https://github.com/potentialventures/cocotb.git
     
+    # Install cocotb
+    python setup.py install
```

Comment:
I'd recommend `pip install ./cocotb` instead, [since then the user stands a chance at uninstalling it](https://stackoverflow.com/a/15731459/102441).

At the least, you're missing a `cd` or `cocotb/setup.py` here.

Reply:
Ohh my. Wrong directory.  Thank you.